### DISCUSSION TITLE
Is there a way to store a file on a single node/rack/dc (how to prevent files from being chunked into different locations)

### DISCUSSION THREAD

**Discussion by maelle-le-bon** [seaweedfs/seaweedfs#5056] [Q&A]
Hello,
Thank you very much for this, I am playing with seaweed FS for a week, and I am very impressed of performances.
I use it as distributed storage without replication to store files from 1GB up to 10GB. My 3 servers are hosted at different locations and they sometimes go offline for maintenance.

**Issue:**
When I upload a file, I can see the directory size increasing. But when the file is too big, some chunks are stored at different locations.
For example : With a 5GB file, the beginning of the file seems to be stored on server A, while the end seems to be stored on server B.

**Goal:**
My goal is to store the entire file on a single server. I'd like to be stored this way to allow one of my 3 servers to be offline while keeping ~66% of my files to be fully available.

Is there a way to instruct the master/filer to store each file on a single server ?

**Comment by nextbiz938** [comment]
I have the same question.

**Comment by chrislusf** [comment]
In https://github.com/seaweedfs/seaweedfs/wiki/Filer-Server-API there are options to specify dataCenter, rack, dataNode
